pkg/registry/core/service/registry.go

```diff
@@ -58,7 +58,7 @@ func (s *storage) ListServices(ctx genericapirequest.Context, options *metainter
 }
 
 func (s *storage) CreateService(ctx genericapirequest.Context, svc *api.Service) (*api.Service, error) {
```

Comment:
only safe due to usage.  Issue to move this one to point of use.

Reply:
This is actually used. Wired through the createValidation from service REST Create.